test/geometry_tests.cpp

```diff
@@ -162,6 +204,48 @@ TEST(Point2DPlanning_Griz_Test, bresenham_conversion_octant1_test) {
 
 }
 
+TEST(Point2DPlanning_Griz_Test, raytraceLine_conversion_pointTest) {
+	Cell startPix = {.x = 0, .y = 0};
```

Comment:
I didn't know about the `.member` syntax. Though, you don't really need to do that as
`const Cell start {0, 0}` will suffice.